analytics/views.py

```diff
@@ -490,6 +492,44 @@ def realm_summary_table(realm_minutes: Dict[str, float]) -> str:
         except Exception:
             row['history'] = ''
 
+    # get the yearly revenue
+    total_amount = 0
+    if settings.BILLING_ENABLED and settings.CORPORATE_ENABLED:
+        from corporate.lib.stripe import extract_current_subscription, extract_percent_off
+        from corporate.models import Customer
+        stripe.api_key = get_secret('stripe_secret_key')
+        stripe_customer_to_plan = {}
+
+        try:
+            stripe_customers = stripe.Customer.list(limit=100)
+            for stripe_customer in stripe_customers:
+                subscription = extract_current_subscription(stripe_customer)
+                if subscription is not None:
+                    percent_off = extract_percent_off(stripe_customer)
+                    amount = (subscription.plan.amount * (100 - percent_off)/100)
+                    stripe_customer_to_plan[stripe_customer.id] = {
+                        "interval": subscription.plan.interval,
+                        "amount": amount/100,
```

Comment:
I suppose if we wanted cents, we'd put this as `amount / 100.0`?  Maybe worth a comment, but we probably do just want whole-dollar rounded amounts for this.

Reply:
yeah, no cents (these are all approximations anyway)

Reply:
Yeah, sgtm, don't want the `.0` part if we're not bothering with cents for this page.